cloudinit/util.py

```diff
@@ -600,6 +605,11 @@ def get_linux_distro():
     distro_version = ''
     flavor = ''
     os_release = {}
+    if is_FreeBSD():
+        distro_name = 'freebsd'
+        distro_version = subp(['/usr/bin/uname', '-r'])
```

Comment:
i've also changed the call to `freebsd-version`, which is a shell script, to `uname -r` which is a compiled binary.

Reply:
On linux, we expect PATH to be sane, so we avoid specifying paths to binaries.  Is /usr/bin not typically in the path for root or other users?